styles/mob.css

```diff
@@ -33,6 +33,8 @@
     margin-bottom: 1em;
     height: 0;
     overflow:hidden;
+    transition-duration: 500ms;
+    transition-timing-function: ease-in-out;

```

Comment:
Put your transition declaration in the rule that is for the element in its default state, otherwise the animation only gets applied when going from the 'changed' state back to its default state